node/task.ts

```diff
@@ -588,6 +588,10 @@ export function exist(path: string): boolean {
     return exist;
 }
 
+export function writeFileSync(file: string, data: string, options? : any) {
```

Comment:
technically we can do data: string | Buffer
Buffer is defined in node and we already leak node types (FsStats)